community-handbook/community-report.md

```diff
@@ -0,0 +1,33 @@
+# Community Report
+
+The CHAOSS Community Report is a free service that the CHAOSS community provides to open source communities.
```

Comment:
@germonprez because this is a joint effort with the LF do we need to explicitly say that here? Or is it ok to just mention CHAOSS?